lib/processes/frbPostProcess.cpp

```diff
@@ -205,24 +238,32 @@ void frbPostProcess::main_thread() {
                         frb_header_offset[b * _num_gpus + thread_id] = ofs;
```

Comment:
In the two lines above it looks like `scl` could be zero in the case all packets are lost, if that happened there will be a divide by zero error.  Also in that case I think the header should just have `scale` and `offset` set to `0,0` or `1,0` if that makes more sense to L1.  Since in this case those values are otherwise undefined.